lib/membrane/rtp/utils.ex

```diff
@@ -29,4 +29,24 @@ defmodule Membrane.RTP.Utils do
         {<<payload::binary, 0::size(zeros_no)-unit(8), padding_size>>, padding_size}
     end
   end
+
+  @spec from_which_cycle(number() | nil, number(), number()) :: :current | :previous | :next
+  def from_which_cycle(nil, _new, _cycle_length), do: :current
```

Comment:
can you make a function header without definition that will handle the function spec? Something like this:
```elixir
@spec from_which_cycle(number() | nil, number(), number()) :: :current | :previous | :next
def from_which_cycle(previous_value, new_value, cycle_length)
```

And after that you can do the patter matching on the function. Right now it is difficult to read the function definition when the first pattern match is `nil` and there is no further information until I look at the next match what the first param really is. And it is no necessarily `base` :)

Reply:
Done